wg21_paper_tracker/tests/test_pipeline.py

```
@@ -0,0 +1,166 @@
+    resp.headers = {"content-type": "application/pdf"}
+    resp.iter_content = lambda chunk_size: (b"\x25\x50\x44\x46",)
+    with patch("wg21_paper_tracker.pipeline.requests.get", return_value=resp):
+        result = _download_file(url, filepath)
+    assert result is True
+    assert filepath.read_bytes() == b"\x25\x50\x44\x46"
+
+
+def test_download_file_uses_timeout():
+    """_download_file calls requests.get with DOWNLOAD_TIMEOUT."""
+    url = "https://example.com/f"
+    filepath = Path("/tmp/out")
```

Ruff (0.15.5)

[error] Probable insecure usage of temporary file or directory: "/tmp/out"

(S108)